src/mesh/difops.cxx

```diff
@@ -281,7 +283,7 @@ const Field3D Grad_par_CtoL(const Field3D &var) {
       }
     }
 
-    result = mesh->fromFieldAligned(result);
+    result = mesh->fromFieldAligned(result_fa);
```

Comment:
I'm not sure that `result_fa` is actually set in the above loop. What happened previously was that `result` was used to store the field aligned result and then we did an "in-place" transform back into non-field aligned : `result = mesh->fromFieldAligned(result);`

Reply:
Cheers @d7919, that was just a mistake.